test/scenarios/bookshop/funcs.test.js

```diff
@@ -155,6 +155,82 @@ describe('Bookshop - Functions', () => {
   })
 
   describe('Date and Time Functions', () => {
+
+    const testCases = [
+      { value: '1970-03-30T01:01:01.000Z', expectedResultPostgres: 5, expectedResult: 5 },
+      { value: 'foo', expectedResultPostgres: `invalid input syntax for type timestamp: "foo"`, expectedResult: 0, },
+    ]
+
+    const testCasesYearMonthDay = [
+      { value: '1970-03-30', expectedResultPostgres: 5, expectedResult: 5 },
+      { value: '03/30/1970', expectedResultPostgres: 5, expectedResult: 0 },
+      { value: '03.30.1970', expectedResultPostgres: 5, expectedResult: 0 },
+      { value: '01:01:01', expectedResultPostgres: `Is time but date expected`, expectedResult: 0 },
```

Comment:
`expectedResultPostgres` is against the fundamental purpose of having the new db service. Where everything behaves the same. The error message might differ, but just don't expect them to exactly match. Just expect them to fail or succeed.

Reply:
But for example: when i send `where day('foo') = 30` with postgres the request runs into the catch block. When i send it without postgres i just get back an empty array (request succeeded). How can i check without checking on the db.

Reply:
The problem is that the databases respond differently. So rather then changing the expectation in the test. It is required to adjust the behavior to be aligned.